manual/src/main/asciidoc/user-guide/deploy.adoc

```diff
@@ -33,6 +33,12 @@ Now you have Cellar features available in your Karaf instance:
 karaf@root()> feature:list |grep -i cellar
 ----
 
+[NOTE]
+====
+If you have built Cellar from sources, you have to specify the build version.
+Karaf will install your local build version of Cellar.
+====
+
```

Comment:
If you don't change the version, you can still use:

```
karaf@root()> feature:repo-add cellar
```

It will work. So I don't really understand the purpose of this note. Can you elaborate and eventually provide an example please ?